cmd/ddev/cmd/root.go

```diff
@@ -125,3 +117,7 @@ func dockerNetworkPreRun() {
 		util.Failed("Unable to create/ensure docker network %s, error: %v", netName, err)
 	}
 }
+
+func init() {
+	RootCmd.PersistentFlags().BoolVarP(&util.JSONOutput, "json-output", "j", false, "If true, user-oriented output will be in JSON format.")
```

Comment:
It would be nice to leave the door open to future output formats. I'd probably prefer one of the following methods here: 

- Drush style: `--format=json`
- kubectl/helm golang style: `-o/--output=json`